### DISCUSSION TITLE
Access logs?

### DISCUSSION THREAD

**Discussion by bfors-rvoh** [Arize-ai/phoenix#5272] [Q&A]
Hello,
I'm glad to see the inclusion of auth in the 5.0 release. As part of this feature, is there now a supported way to capture user activity logs? I mainly want to store which projects/traces a user has accessed.

Thanks,
Brian

**Comment by RogerHYang** [comment]
This is not implemented. Would you mind opening an [enhancement request](https://github.com/Arize-ai/phoenix/issues/new?assignees=&labels=enhancement%2Ctriage&projects=&template=feature_request.md&title=%5BENHANCEMENT%5D) to let us know the details of your requirements?
